src/main/webapp/app/guided-tour/guided-tour.service.ts

```diff
@@ -142,21 +142,15 @@ export class GuidedTourService {
     /**
      * Trigger callback method if there is one and finish the current guided tour by updating the guided tour settings in the database
      * and calling the reset tour method to remove current tour elements
-     *
      */
-    private finishGuidedTour() {
+    public finishGuidedTour() {
```

Comment:
I set this one to public since I wanted to write a tests for that.

Reply:
It's also possible (if not ideal) to leave it private and suppress the type error in the test file.

Reply:
Also, you could access the private method in the test like so:
```ts
object['finishGuidedTour']()
```